docs/safe_lambda_deployments.rst

```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
 ---------------------------------------------
 
 Every Lambda function can have any number of Versions and Aliases
-associated with them. Versions are immutable snapshot of function
+associated with them. Versions are immutable snapshots of a function
 including code & configuration. If you are familiar with git, they are
-similar to commits. It is a good practice in general to publish a new
+similar to commits. In general, it is a good practice to publish a new
 version every time you update your function code. When you invoke a
-specific version (using function name + version number combination) you
-are guaranteed to get the same code & configuration irrespective of
+specific version (using the function name + version number combination) you
+are guaranteed to get the same code & configuration irrespective of the
 state of the function. This protects you against accidentally updating
 production code.
 
 To effectively use the versions, you should create an Alias which is
```

Comment:
To effectively use versions you can create an alias which will act as a pointer to a specific version

Reply:
They aren't 'literally' a pointer... It's frustrating that the second definition of literally is figuratively 
https://www.merriam-webster.com/dictionary/literally

Reply:
So... @michaeldclifford is literally correct? ;)